project/CopyrightHeader.scala

```diff
@@ -11,22 +11,18 @@
  * Copyright (C) 2018-2022 Lightbend Inc. <https://www.lightbend.com>
  */
 
-import de.heikoseeberger.sbtheader.HeaderPlugin.autoImport._
 import de.heikoseeberger.sbtheader.{ CommentCreator, HeaderPlugin, NewLine }
 import org.apache.commons.lang3.StringUtils
-import sbt.Keys._
 import sbt._
 
-import MultiJvmPlugin.MultiJvmKeys._
+object CopyrightHeader extends AutoPlugin {
+  import HeaderPlugin.autoImport._
 
-trait CopyrightHeader extends AutoPlugin {
+  override def requires = HeaderPlugin
+  override def trigger = allRequirements
 
-  override def requires: Plugins = HeaderPlugin
-
-  override def trigger: PluginTrigger = allRequirements
-
-  protected def headerMappingSettings: Seq[Def.Setting[_]] =
-    Seq(Compile, Test, MultiJvm).flatMap { config =>
+  override def projectSettings = Def.settings(
+    Seq(Compile, Test).flatMap { config =>
```

Comment:
Will it now update the header in multi-node folder too?

Reply:
Yes, please see https://github.com/apache/incubator-pekko/pull/1005#issue-2092289217 specifically

> You can test this by manually removing a header from a source file and then doing remote-tests/headerCheckAll (using remote-tests as an example). I did this myself but best to double check.